### ISSUE TITLE
Fly out categories menu for Bootstrap

### ISSUE THREAD

**Issue by scottcwilson** [lat9/ZCA-Bootstrap-Template#415]
What are your thoughts about adding a flyout menu for this template - some of the older responsive templates had this sort of thing.

![flyout](https://github.com/user-attachments/assets/ed2d3a8f-0243-4608-b181-bdbfa5f3dd2b)

**Comment by torvista** [comment]
Yes. I would consider it pretty standard functionality...along with dropdowns for the category_tabs.

**Comment by lat9** [comment]
Perhaps start it out as a plugin to iron any issues out.

**Comment by torvista** [comment]
At the moment it's a half-working dogs dinner, cobbled from multiple examples... l need to start again...

**Comment by torvista** [comment]
https://github.com/zencart/zencart/discussions/6639

**Comment by lat9** [comment]
@torvista, let's start this out as a plugin for Bootstrap rather than incorporating into the template itself.

**Comment by scottcwilson** [comment]
I'd love to see a bunch of Bootstrap plugins and this is a great start.

**Comment by torvista** [comment]
I did mods to the ul generator to get my efforts working (not saying it was necessary)....and I spotted a lurking Dr.Byte branch for that ... hence all part of my prod to have a discussion first, I imagine I'm not the first to have attempted this.

**Comment by barco57** [comment]
"along with dropdowns for the category_tabs." - yes

**Comment by scottcwilson** [comment]
@torvista Can you release this as a plugin?

**Comment by torvista** [comment]
I abandoned my attempt to develope a BS template for my site...could never this aspect to work right.
Pxssing in the wind trying to second guess by trial and error why multiple layers of css don't work as expected drives me insane.
Give me logic any day.
